app/interceptors/ExceptionInterceptor.java

```diff
@@ -57,8 +53,6 @@ public Object invoke(MethodInvocation method) throws Throwable {
         try {
             return method.proceed();
         } catch(Throwable throwable) {
-            throwable = Throwables.getRootCause(throwable);
```

Comment:
The current behavior looks at the root cause to decide whether to log it or not, which I think is bug.  So this change makes total sense.  In the meantime, logger.debug(message, throwable) downstream logs the entire stack trace to the root cause, which is the desired behavior.  When we look at the logs for errors we do want to know the root cause.  Overall this is a nice fix.